dependencies/go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/google/docsy/dependencies
 
 go 1.12
+
+require (
+	github.com/FortAwesome/Font-Awesome v0.0.0-20210804190922-7d3d774145ac // indirect
+	github.com/twbs/bootstrap v4.6.1+incompatible // indirect
+)
```

Comment:
- I'm curious why this wasn't needed before?
- Also, we now have BS and FA dependencies in two places. Is this what we want?
  - In this file
  - The top-level, https://github.com/google/docsy/blob/df32a130b47382090b8b0869743c5b3d0b2f55b4/go.mod#L5-L8

Reply:
> * I'm curious why this wasn't needed before?

I realized that adding these lines makes the installation process more fail safe, that's why I added them.

>     * Also, we now have BS and FA dependencies in two places. Is this what we want?

Yes. I just made a test and removed BS + FA dependencies from `go.mod` at root level. This immediately broke installation, so I want to keep everything as is.